packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.constants.js

```diff
@@ -2,6 +2,8 @@ export const DATACENTER_CONFIGURATION_KEY = 'dc';
 export const ENGINE_CONFIGURATION_KEY = 'engine';
 export const PLAN_CODE_TEMPLATE = 'cloud-db-xxxx-instance';
 export const PRODUCT_NAME = 'cloudDB';
+export const WEB_HOSTING_PRODUCT_NAME = 'webHosting';
+export const PRESELECTED_DB_CATEGORY = 'private';
```

Comment:
Why to add this in this file, you have the constant file nearest.

Reply:
PRESELECTED_DB_CATEGORY  was used for preselectDbCategory  prop.

But if we don't set it, we don't need it anymore :)

Reply:
same as https://github.com/ovh/manager/pull/10920#discussion_r1494037892